bundles/org.openhab.binding.emotiva/src/main/java/org/openhab/binding/emotiva/internal/EmotivaUdpSendingService.java

```diff
@@ -122,12 +123,11 @@ public void connect(Consumer<EmotivaUdpResponse> listener, boolean logNotThrowEx
         }
     }
 
-    private void handleReceivedData(DatagramPacket answer, byte[] receivedData,
-            Consumer<EmotivaUdpResponse> localListener) {
+    private void handleReceivedData(DatagramPacket answer, Consumer<EmotivaUdpResponse> localListener) {
         // log & notify listener in new thread (so that listener loop continues immediately)
         executorService.execute(() -> {
             if (answer.getAddress() != null && answer.getLength() > 0) {
-                logger.trace("Received data on port '{}': {}", answer.getPort(), receivedData);
+                logger.trace("Received data on port '{}'", answer.getPort());
```

Comment:
```suggestion
                logger.debug("Received data on port '{}'", answer.getPort());
```
As it no longer has contents, it can be changed to debug